beacon_node/beacon_chain/src/validator_custody.rs

```diff
@@ -0,0 +1,440 @@
+use std::{
+    collections::{BTreeMap, HashMap},
+    sync::atomic::{AtomicU64, Ordering},
+};
+
+use parking_lot::RwLock;
+
+use ssz_derive::{Decode, Encode};
+use types::{ChainSpec, Epoch, EthSpec, Slot};
+
+/// A delay before making the CGC change effective to the data availability checker.
+const CUSTODY_CHANGE_DA_EFFECTIVE_DELAY_SECONDS: u64 = 30;
```

Comment:
can't we have this delay in slots as that's what how this constant is used below?

Reply:
Discussed in person - using seconds so the delay is consistent on devnets with low slots per sec as well.